components/Players/DraggableQueue.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-list>
+    <draggable v-model="queue" v-bind="dragOptions" class="list-group">
+      <v-hover
+        v-for="(item, index) in queue"
+        :key="`${item.Id}-${getUuid()}`"
+        v-slot="{ hover }"
+        class="pa-0 ma-0"
+      >
+        <v-list-item ripple class="pa-0 ma-0" @click="onClick(index)">
+          <v-list-item-action
+            v-if="!hover"
+            class="list-group-item d-flex justify-center d-flex text-caption"
+            :class="{ 'primary--text': isPlaying(index) }"
+          >
+            {{ index }}
+          </v-list-item-action>
+          <v-list-item-action v-else class="justify-center d-flex">
+            <v-icon>mdi-drag-horizontal</v-icon>
+          </v-list-item-action>
+          <v-list-item-avatar tile class="list-group-item">
+            <blurhash-image :item="item" />
+          </v-list-item-avatar>
+
+          <v-list-item-content>
+            <v-list-item-title
+              :class="{ 'primary--text': isPlaying(index) }"
+              class="text-truncate ml-2 list-group-item"
+            >
+              {{ item.Name }}
+            </v-list-item-title>
+            <v-list-item-subtitle
+              v-if="getArtists(item)"
+              class="ml-2 list-group-item"
+              :class="{ 'primary--text': isPlaying(index) }"
+            >
+              {{ getArtists(item) }}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+
+          <v-list-item-action>
+            <favorite-button :item="item" />
+          </v-list-item-action>
+          <v-list-item-action class="mr-2">
+            <item-menu :item="item" />
+          </v-list-item-action>
```

Comment:
These are superfluous, since both of these are already right below the dialog.

Reply:
Disabled them for the current playing item